### DISCUSSION TITLE
keda-operator memorypressure

### DISCUSSION THREAD

**Discussion by hila3000** [kedacore/keda#1032] [General] [answered]
Hi,

We're using keda in Production.
I've noticed keda-operator pods are being evicted due to memory pressure.
When checking the graphs, I see that we have some spikes of memory - usually consumption is on 0mb, but the spikes reach up to 10GB.

My question is-
Is it reasonable for keda-operator to reach 10GB of memory consumption?
Anything in configuration I should set to avoid that if it's not a valid consumption (like requests and limit resources settings in the keda-deployment)?

Thanks,
Hila

**Comment by ahmelsayed** [comment]
It probably shouldn't need 10GB. which scalers are you using? and is there any particular pattern you can spot when these leaks happen?

**Comment by hila3000** [comment, marked as answer]
Version 1.5.0 wasn't closing stackdriver client connections. That's why it happened.
I opened an issue about it, they released a version for me to test and now connections are getting closed and memory usage is really low and steady.
